.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release
+
+on:
+  create:
+    ref_type: tag
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: github.repository == 'voxpupuli/facterdb'
```

Comment:
Something I learned after introducing this is that there's also a variable `repository_owner` which in this case is `voxpupuli`. That brings our actions closer together and probably allows you to blindly copy-paste them between our gems.